src/document/DocumentAnnotator.ts

```diff
@@ -93,42 +93,16 @@ export default class DocumentAnnotator extends BaseAnnotator {
 
         const annotation = getAnnotation(this.store.getState(), annotationId);
         const annotationPage = annotation?.target.location.value;
+        const annotationPageEl = this.getPage(annotationPage || 1);
```

Comment:
Would it make sense for L95 to be `annotation?.target.location.value ?? 1`?

Reply:
Sure.